matching-notification-api/src/main/java/org/phenotips/matchingnotification/match/internal/DefaultPatientMatch.java

```diff
@@ -714,6 +693,7 @@ public JSONObject getNotificationHistory()
     public void setNotificationHistory(JSONObject notificationHistory)
     {
         this.notificationHistory = (notificationHistory != null) ? notificationHistory.toString() : null;
+        this.notifiedTimestamp = new Timestamp(System.currentTimeMillis());
```

Comment:
wring change, `notifiedTimestamp` with be set to current time each time we call `setNotificationHistory()` in `preserveOriginalMatchMetaInfo()`

Reply:
I think that is OK. It is not really used anywhere (we have notification history if we need details), and in the next update we should probably remove it completely